bigtop-packages/src/common/ambari/patch14-support-openeuler.diff

```diff
@@ -0,0 +1,97 @@
+diff -Naur apache-ambari-2.7.5-src/ambari-common/src/main/python/ambari_commons/os_check.py apache-ambari-2.7.5-src-openEuler/ambari-common/src/main/python/ambari_commons/os_check.py
```

Comment:
This patch file would be applied to the other Distros.
But the other Distros don't need this patch file.

Reply:
> This patch file would be applied to the other Distros. But the other Distros don't need this patch file.

yes, ambari is not support openeuler, this patch is we add openuler code to supoort ambari.
1.  we run smoketest for other Distros  ,it not unaffected.
2.  if you want not add this patch in bigtop-packages/src/common/ambari/ ,Where should we place?
     add sed command in do-component-build ?
@guyuqi

Reply:
@uscxiexing
Thanks for your efforts and elaboration.

>>if you want not add this patch in bigtop-packages/src/common/ambari/ ,Where should we place?
add sed command in do-component-build ?

Yes, it's a little bit tricky. 
If you kindly tested it on other Distros(Centos,Debian,Ubuntu ,Fedora, Rockylinux), 
we could temporarily put this patch file here (`bigtop-packages/src/common/ambari`).
I think we could remove this patch file after you post this patch in Ambari in the future.